### PR TITLE
Design review updates 

### LINKED ISSUE #3609: Decrease size of core stats bar
Link to frame
https://www.figma.com/file/bZh7tOiLm6Ls9Hl4RL6qMC5X/Augur-Redesign?node-id=635%3A30189

The default state would look like this but we need to finish the active states for each menu item. For now lets just make the size changes and anything else that can be done for now

### PR DESCRIPTION
**Pull request by andrewdamelio**
Closes #3609, #3608 and #3607

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🆒

 This link will always point to this PR's latest changes: https://peek.run/36whq5o

**Comment by andrewdamelio** [comment]
@matt-bullock Can you give these changes a quick peak https://peek.run/36whq5o 
in reference to #3609, #3608 and #3607

File: packages/augur-ui/src/modules/app/components/top-bar.styles.less [hunk 1]
```diff
@@ -36,41 +61,68 @@
...
 }
 
-.statsContainer {
-  width: min-content;
+.Logo {
 
-  @media @breakpoint-mobile {
-    width: auto;
+  div {
+    width: 24px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/top-bar.styles.less]: Use size constant here

File: packages/augur-ui/src/modules/app/components/top-bar.styles.less [hunk 2]
```diff
@@ -36,41 +61,68 @@
...
+    margin-left: 3rem;
+    margin-right: auto;
 
-  > label {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    margin: 0;
+    div {
+      width: 40px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/top-bar.styles.less]: Keep up the trend and use size constant here

File: packages/augur-ui/src/modules/filter-sort/components/filter-dropdowns.styles.less [hunk 3]
```diff
@@ -5,16 +5,10 @@
...
-    justify-content: space-between;
-    margin-bottom: @size-16;
-  }
-
   > div {
-    margin-bottom: @size-16;
+    // margin-bottom: @size-16;
     min-width: 250px;
-    height: 44px;
+    height: 36px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/filter-sort/components/filter-dropdowns.styles.less]: Size constant to keep with thetrend

File: packages/augur-ui/src/modules/markets-list/components/markets-header.styles.less [hunk 4]
```diff
@@ -74,11 +75,11 @@
   > div > article {
     flex: 1;
     margin-bottom: @size-16;
+    height: 44px;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/markets-list/components/markets-header.styles.less:78]: I think 40 is the largest size we got so this is fine

**Review by bthaile — APPROVED** [review]
Lgtm, some not pic with using size constants

**Review by matt-bullock — COMMENTED** [review]
funds should be 2 decimal places: $5000.00
<details>

    Page path:     /#!/markets
    Browser:       Chrome 76.0.3809.132 / Mac OS 10.14.6
    Viewport:      1112 x 884 @2x
    Language:      en-GB
    Cookies:       Yes

</details>

[View on FeaturePeek](https://dashboard.featurepeek.com/peek/36whq5o#/#!/markets)

**Pull request merged**